packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -379,6 +410,11 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
                 }
                 this.setState({ isOpen: false });
             } else if (!(e.key === "Backspace" || e.key === "ArrowLeft" || e.key === "ArrowRight")) {
+                // Custom target might not be an input, so certain keystrokes might have other effects
+                // (space pushing the scrollview down), prevent default events before opening the popover
+                if (this.props.customTarget != null && (e.key === " " || e.key === "ArrowDown")) {
+                    e.preventDefault();
+                }
```

Comment:
could we instead make this:
- only open the popover on " " or "Enter"
    - thinking this is different for custom target because it's an element that opens the popover (on keyboard action), not something a user should be typing in (could re-think if custom targets need to actually be inputs but I wouldn't really expect this)
- not prevent default on `ArrowDown`
    - when focusing on a native button and hitting down the page scrolls, so I think that's okay/expected